backend/src/controllers/DailyIntakeController.js

```diff
@@ -0,0 +1,76 @@
+const db = require("../database/database");
+
+
+const insertIntake = async (req, res) => {
+    const userId = req.user.data.user_id;
+    const {intakeID, foodID} = req.body;
+    const currentDate = new Date();
+    try {
+        const insertedIntake = await db.one(
+            "INSERT INTO dailyIntake(daily_intake_id, user_id, food_id, date) VALUES($1, $2, $3, $4) RETURNING *",
+            [intakeID, userId, foodID, currentDate]
+        );
+        res.status(200).send(insertedIntake);
```

Comment:
- We don't need `intakeID` because that is `serial` type meaning it is auto incremented 
- Take out `intakeID` in both line 6 and line 9-10